src/stores/JsonViewerStore.ts

```diff
@@ -49,6 +51,7 @@ export const createJsonViewerStore = <T = unknown> (props: JsonViewerProps<T>) =
   return create<JsonViewerState>()((set, get) => ({
     // provided by user
     enableClipboard: props.enableClipboard ?? true,
+    enableHighlight: props.enableHighlight ?? true,
```

Comment:
I think this feature should be disabled by default.
```suggestion
    enableHighlight: props.enableHighlight ?? false,
```